### ISSUE TITLE
Include Logits in Output Class

### ISSUE THREAD

**Issue by LazerLambda** [LazerLambda/Promptzl#81]
- Add working function for returning logits from classify method
